src/test/test-path-util.c

```diff
@@ -511,7 +511,24 @@ static void test_hidden_or_backup_file(void) {
         assert_se(!hidden_or_backup_file("test.dpkg-old.foo"));
 }
 
+static void test_systemd_installation_has_version(const char *path) {
+        int r;
+        const unsigned versions[] = {0, 231, atoi(PACKAGE_VERSION), 999};
+        unsigned i;
+
+        for (i = 0; i < ELEMENTSOF(versions); i++) {
+                r = systemd_installation_has_version(path, versions[i]);
```

Comment:
hmm, given that s_i_h_v() returns a bool, I'd probably not reuse "r" here, but instead write:

```c
for (…) {
        bool b;
        b = s_i_h_v(…);
        …
}
```

i.e.avoid the implicit bool → int case...